packages/groonga/build.sh

```diff
@@ -0,0 +1,22 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/groonga/groonga/
+TERMUX_PKG_DESCRIPTION="An embeddable fulltext search engine"
+TERMUX_PKG_LICENSE="LGPL-2.1"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION="14.0.6"
+TERMUX_PKG_SRCURL=https://github.com/groonga/groonga/releases/download/v${TERMUX_PKG_VERSION}/groonga-${TERMUX_PKG_VERSION}.tar.gz
+TERMUX_PKG_SHA256=d5c693400c9cbe91b63e0b5bc489f9f07af1bd68f12a089ccd1482ef69d6cc6c
+TERMUX_PKG_AUTO_UPDATE=true
+TERMUX_PKG_FORCE_CMAKE=true
+TERMUX_PKG_DEPENDS="libandroid-execinfo, libarrow-cpp, libc++, liblz4, libstemmer, onigmo, xxhash, zlib, zstd"
+TERMUX_PKG_BUILD_DEPEPNDS="rapidjson"
+TERMUX_PKG_EXTRA_CONFIGURE_ARGS="
+-DGRN_WITH_APACHE_ARROW=ON
+-DGRN_WITH_LIBSTEMMER=system
+-DGRN_WITH_LZ4=system
+-DGRN_WITH_ONIGMO=ON
```

Comment:
We can remove this because this is the default.

Reply:
I agree. Adding the option explicitly will help to enable/disable it easily. The rule is followed in other build scripts also. 
_Explicit is better than implicit._ 🐍